data/858/984/01/85898401.geojson

```diff
@@ -38,6 +38,9 @@
     "name:eng_x_variant":[
         "Suessenbruenn"
     ],
+    "name:ger_x_preferred_x_variant":[
```

Comment:
This property name is 2x jumbled.

Reply:
This should be fixed in the most recent commit.